### ISSUE TITLE
value es solo para inputs

### ISSUE THREAD

**Issue by linkstrifer** [sergioandresco/Calculadora-Cliente-Servidor#2]
https://github.com/sergioandresco/Calculadora-Cliente-Servidor/blob/d91a6317dec6d56211aff95f8fc9322299f4995c/Cliente/Index.html#L59

Si quieres cambiar el contenido de un elemento que no es `input`, usa `textContent`, quedaría así:

```js
pantalla1.textContent = pantalla1.textContent + numero;
```

**Comment by sergioandresco** [comment]
Efectivamente, muchas gracias Link, tienes toda la razón, ya me funciono 🙌🏼
